### PR TITLE
contrib: fix secadvisor default config

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-compile-tests
run skydive-functional-hw-tests
run skydive-cdd-overview-tests

**Comment by safchain** [comment]
run skydive-k8s-tests
run skydive-unit-tests

File: contrib/pipelines/secadvisor/secadvisor.yml.default [hunk 1]
```diff
@@ -1,4 +1,6 @@
 host_id: ""
+analyzers:
```
Comment (safchain) [inline contrib/pipelines/secadvisor/secadvisor.yml.default]: I do not see where this is supposed to be used :(

Comment (hunchback) [reply contrib/pipelines/secadvisor/secadvisor.yml.default]: used here: https://github.com/skydive-project/skydive/blob/960a50d2c797c60fa3c656d8335dd7406ec8dd07/contrib/pipelines/secadvisor/transform.go#L97

**Comment by hunchback** [comment]
@safchain - I prefer the approach of getting all the skydive defaults - see fix in #1914

**Pull request closed without merging**